runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/FakeWindmillServer.java

```diff
@@ -211,7 +211,11 @@ public CommitWorkResponse commitWork(Windmill.CommitWorkRequest request) {
     validateCommitWorkRequest(request);
     for (ComputationCommitWorkRequest computationRequest : request.getRequestsList()) {
       for (WorkItemCommitRequest commit : computationRequest.getRequestsList()) {
-        commitsReceived.put(commit.getWorkToken(), commit);
+        // Throw away per work item latency attributions because they are not deterministic in tests
```

Comment:
should we have an option or separate method to do this after pulling off of commitsReceived?
That would let us have a test that verified that things were reasonable.